controller-vtep/vtep.c

```diff
@@ -366,7 +366,11 @@ vtep_macs_run(struct ovsdb_idl_txn *vtep_idl_txn, struct shash *ucast_macs_rmts,
         for (i = 0; i < port_binding_rec->n_mac; i++) {
             const struct vteprec_ucast_macs_remote *umr;
             const struct sbrec_port_binding *conflict;
-            char *mac = port_binding_rec->mac[i];
+            char *mac = xstrdup(port_binding_rec->mac[i]);
+            char *p = strchr(mac, ' ');
```

Comment:
Instead of this, I would suggest to use the ovn-util helper function - extract_addresses() to get the mac from the port_binding->mac column.

something like

***
struct lport_addresses laddrs;
if (!extract_addresses(port_binding_rec->mac[i], &laddrs)) {
    continue;
}

..

The util  function is here - https://github.com/ovn-org/ovn/blob/master/lib/ovn-util.c#L182